python/lsst/afw/cameraGeom/utils.py

```diff
@@ -73,17 +82,29 @@ def prepareWcsData(wcs, amp, isTrimmed=True):
 
 def plotFocalPlane(camera, fieldSizeDeg_x=0, fieldSizeDeg_y=None, dx=0.1, dy=0.1, figsize=(10., 10.),
```

Comment:
Why are the defaults for `fieldSizeDeg_x` and `fieldSizeDeg_y` different?

Why are there underscores in the names?

Reply:
I have no idea about either of those comments, I'm just bringing the documentation up to standard. Do you know why?

Reply:
1. git blame reveals that @kfindeisen added the arguments (and the underscores)
2. The `_y` default is `None` so that it defaults to the value provided for `_x`; seems reasonable

Reply:
I did not add the arguments, I merely renamed them from `pupil*` to `field*`. I can trace them back as far as @SimonKrughoff [in 2014](https://github.com/lsst/afw/commit/d21f31412c45938442cc8d43e5cfc79b2e85f05f#diff-df14be5358d8e32f391a0af5848d7954R60), they may be even older.

In any case, I have no objections to these names being brought up to convention if somebody feels it's within scope.